src/components/pages/TestSuites/TestSuiteDetails/TestSuiteSettings/SettingsTests/Nodes/StepNode.tsx

```diff
@@ -39,7 +42,12 @@ const StepNode: React.FC<StepNodeProps> = props => {
             </Text>
           </TestNodeNameContainer>
         </Tooltip>
-        <DotsDropdown items={[{key: 1, label: <span onClick={() => data.deleteNode(id, data.group)}>Delete</span>}]} />
+        <DotsDropdown
+          items={[
+            {key: 1, label: <span onClick={() => dashboardNavigate('test')}>Configure this test</span>},
```

Comment:
Is it correct URL? `/tests/${test}/settings/test`

Reply:
yes, we want to redirect to the test definition tab